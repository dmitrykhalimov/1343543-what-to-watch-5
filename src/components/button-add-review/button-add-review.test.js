import React from "react";
import renderer from "react-test-renderer";
import ButtonAddReview from "./button-add-review";
import {MemoryRouter} from 'react-router-dom';

describe(`<ButtonPlay /> render`, () => {
  it(`Should ButtonPlay render correctly`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <ButtonAddReview
              id={null}
            />
          </MemoryRouter>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

