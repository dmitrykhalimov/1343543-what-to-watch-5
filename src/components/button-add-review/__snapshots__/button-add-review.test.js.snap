// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonPlay /> render Should ButtonPlay render correctly 1`] = `
<a
  className="btn movie-card__button"
  href="/null/review"
  onClick={[Function]}
>
  Add review
</a>
`;
